website/src/pages/index.tsx

```diff
@@ -47,17 +92,15 @@ export default function Home() {
 
         <section className="features">
           <h2>
-            <Translate
-              id="homepage.features_title"
-            >
-              {`Features`}
-            </Translate>
+            <Translate id="homepage.features_title">{`Features`}</Translate>
           </h2>
+          {
+            // TIPS: TEXT LABELS MUST BE STATIC
```

Comment:
Could you move this to the `checkBoxes` variable instead?

Reply:
I will try it :)